### DISCUSSION TITLE
Extension activation failed - no such file or directory

### DISCUSSION THREAD

**Discussion by stevenh** [community/community#80763] [Copilot Conversations]
### Select Topic Area

Bug

### Body

I re-enabled copilot VSCode extension today and it fails to run logging:
```
[ERROR] [default] [2023-12-18T10:57:23.861Z] Extension activation failed: "ENOENT: no such file or directory, uv_cwd"
```

**Comment by digitarald** [comment]
If its VS Code, can you make sure the extension and VS Code are updated to the latest version? If it persists, please file an issue in https://github.com/microsoft/vscode-copilot-release

**Comment by stevenh** [comment]
I can say I was up to date on both at the time of posting, [possibly related discussion](https://github.com/microsoft/vscode/issues/194458) but originally posted as different initial symptoms.
